consensus/safety-rules/src/safety_rules.rs

```diff
@@ -150,6 +150,28 @@ impl SafetyRules {
         updated
     }
 
+    pub(crate) fn observe_tc(
```

Comment:
Rename as `verify_and_update_last_vote_round` and do similar things as https://github.com/aptos-labs/aptos-core/blob/satya/order_vote_1/consensus/safety-rules/src/safety_rules.rs#L216

Reply:
I can rename to `verify_and_update_last_timeout_round`. The current code is same as `verify_and_update_last_vote_round`. I just implemented in a more rusty way.

Reply:
Renamed.